Dockerfile

```diff
@@ -76,6 +78,11 @@ RUN set -ex \
     python2 python39 \
     # Packages needed for AWT.
     libXext libXrender libXtst libXi libX11-xcb mesa-libgbm libdrm freetype \
+    # Arch specific installs of libsecret and libsecret-devel
+    && if [[ $(uname -m) == "s390x" ]]; then cd /tmp && curl -sSLO https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-0.20.4-2.fc34.s390x.rpm && curl -sSLO https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-devel-0.20.4-2.fc34.s390x.rpm && microdnf install -y --nodocs dnf && dnf install -y -q --nodocs /tmp/libsecret*.rpm && rm -fr /tmp/libsecret*.rpm; fi \
+    && if [[ $(uname -m) == "ppc64le" ]]; then cd /tmp && curl -sSLO https://rpmfind.net/linux/centos/8-stream/BaseOS/ppc64le/os/Packages/libsecret-devel-0.18.6-1.el8.ppc64le.rpm && microdnf install -y --nodocs dnf && dnf install -y -q --nodocs libsecret /tmp/libsecret*.rpm && rm -fr /tmp/libsecret*.rpm; fi \
+    && if [[ $(uname -m) == "x86_64" ]]; then microdnf install -y --nodocs libsecret; fi \
```

Comment:
Also, no need to have `libsecret` install in line 74 as it installs twice, for the arm-based image it will be better to add new `if` section if needed.